micro_ros_setup/scripts/create_bridge_tools.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+
+# Install Qemu on the host PC.
+sudo apt-get install -y --no-install-recommends qemu-user-static binfmt-support 
+update-binfmts --enable qemu-arm 
+update-binfmts --display qemu-arm
+
+ROOT_DIR=${PWD}
+
+# Delete previous build.
+sudo rm -rf $ROOT_DIR/Micro-ROS-Bridge
+
+#Create the new workspace for the Micro-ROS Agent on the hardware bridge.
+mkdir $ROOT_DIR/Micro-ROS-Bridge
+cd $ROOT_DIR/Micro-ROS-Bridge
+WORK_DIR=${PWD}
+
+# Download the cross-compilation tool
+git clone https://github.com/micro-ROS/ros2-performance
```

Comment:
Why do we need a fork of this repo?

Reply:
On the original repo they offer a docker to cross-compile ROS2, so this fork is a modification to cross-compile the micro-ROS agent.